DuckDuckGo/Preferences/View/PreferencesVPNView.swift

```diff
@@ -55,9 +56,9 @@ extension Preferences {
                 }
                 .padding(.bottom, 12)
 
-                // SECTION: Excluded Sites
+                if model.showExcludedSites && !model.appExclusionsFeatureEnabled {
```

Comment:
Will show the legacy site exclusions section if appropriate and if the new-exclusions flag is off.